src/index.ts

```diff
@@ -48,6 +48,16 @@ export interface Profile {
   stages: Stage[];
 }
 
+export interface Firmware {
+  name: string;
+  hostname: string;
+  firmware: string;
+  color: string;
+  model_version: string;
+  serial: string;
+  mock_response: boolean;
```

Comment:
Mock response is dropped now and should be removed. Rest looks good :)
But this is absolutely the wrong repo. It should be in the API repo instead